src/github.ts

```diff
@@ -0,0 +1,45 @@
+import { Octokit } from "@octokit/rest";
+
+/**
+ * Create a GitHub issue indicating that sync from upstream failed.
+ * @param {Octokit} client the GitHub client to use that is authenticated as the GitHub installation.
+ * @param {string} org the GitHub organization where the issue is to be created.
+ * @param {string} repo the GitHub repository where the issue is to be created.
+ * @param {string} upstreamOrg the upstream GitHub organization that sync was from.
+ * @param {Array<number>} upstreamPRIDs the upstream pull-request IDs that were part of the failed sync.
+ * @param {string} reason the reason that the sync failed.
+ * @param {number} prID an optional pull-request ID of the forked repository's sync PR. This isn't set if the PR
+ *   couldn't be created due to something like a merge conflict.
+ * @return {Promise<number>} a Promise that resolves to the created GitHub issue ID.
+ */
+export async function createFailureIssue(
+  client: Octokit,
+  org: string,
+  repo: string,
+  upstreamOrg: string,
+  upstreamPRIDs: Array<number>,
+  reason: string,
+  prID?: number,
+): Promise<number> {
+  return new Promise((resolve, reject) => {
+    const title = `😿 Failed to sync the upstream PRs: #${upstreamPRIDs.join(", #")}`;
+    const prSuffix = `\n* ${upstreamOrg}/${repo}#`;
+    let body =
+      `The 🤖 sync bot 🤖 failed to sync the following upstream pull-requests because ${reason}:` +
+      `${prSuffix}${upstreamPRIDs.join(prSuffix)}\n\n` +
+      "![sad Yoda](https://media.giphy.com/media/3o7qDK5J5Uerg3atJ6/giphy.gif)";
```

Comment:
Should there be a comment here about closing the issue to ignore it?
And that mirroring is paused until the PR/Issue are resolved?

Reply:
Good point!